server/src/cloud/util.ts

```diff
@@ -1,32 +1,39 @@
 export interface ParseObjectParams {
-    [key: string]: any;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: any;
```

Comment:
Unfortunately not. This is just a utlity type for generic requests on the Cloud function and Parse itself does not have a type for that. So it can be every kind of Object